### DISCUSSION TITLE
Blog Plugin version 9.2.5

### DISCUSSION THREAD

**Discussion by snax44** [squidfunk/mkdocs-material#5928] [Q&A] [answered]
Hi all,

Is it normal that I get the following error when I try to use the plugin "blog" with the version 9.2.5 ?

```bash
(venv) xxxxxxxxxxxxxx#mkdocs serve
ERROR   -  Config value 'plugins': The "blog" plugin is not installed

Aborted with 1 configuration errors!
```

Thanks for any help.
Snax

**Comment by squidfunk** [comment, marked as answer]
Without knowing how you installed Material for MkDocs or what configuration you have, it's not possible for us to help you. The general advice [would be to try to reinstall it](https://squidfunk.github.io/mkdocs-material/getting-started/#installation), because it's likely a hiccup inside your environment ☺️

**Reply by snax44** [reply]
Hi, 
Thanks for your reply.

Just installed it with pip in a venv and add the plugin. No custom settings at all.
I was actually wondering if there is still the limitation for insider.

I'll try to reinstall it.

**Reply by squidfunk** [reply]
The blog plugin is available as part of the community edition since 9.2.0.
